.jenkinsci/linux-build-wheels.groovy

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env groovy
+
+def doPythonWheels() {
+    // version = sh(script: 'git describe --tags \$(git rev-list --tags --max-count=1) || true', returnStdout: true).trim()
+    version = "0.0.1"
+    if (env.GIT_LOCAL_BRANCH != "master") {
+        version += ".dev" + env.BUILD_NUMBER
+    }
+
+    sh(script: "sed -i.bak 's/{{ PYPI_VERSION }}/${version}/g' setup.py;")
+
+    iC = docker.image('quay.io/pypa/manylinux1_x86_64')
+    iC.inside("-v ${WORKSPACE}:/io") {
+        sh(script: '/opt/python/cp35-cp35m/bin/pip wheel --no-deps /io/ -w /io/wheelhouse/', returnStdout: true)
+    }
+}
+
+def publishWheels() {
+    checkTag = sh(script: 'git describe --tags --exact-match ${GIT_COMMIT}', returnStatus: true)
+    withCredentials([usernamePassword(credentialsId: 'ci_nexus', passwordVariable: 'CI_NEXUS_PASSWORD', usernameVariable: 'CI_NEXUS_USERNAME')]) {
+        sh(script: "find wheelhouse -type f -name \"iroha*.whl\" -exec curl -u ${CI_NEXUS_USERNAME}:${CI_NEXUS_PASSWORD} --upload-file {} https://nexus.iroha.tech/repository/artifacts/iroha-python/{} \\;")
```

Comment:
Shall we separate dev builds from master builds on Nexus? I think it is best to have a separate dev folder with the latest build(s?) and latest folder with just a single build from `master` branch that is replaced with each commit